tests/unit/apps/test_core.py

```diff
@@ -825,14 +831,21 @@ def test_nova_compute_get_enable_scheduler_step_no_units(model, units_selected):
         )
         for unit in app.units.values()
     ]
-    assert app._get_enable_scheduler_step(units_selected) == expected_step
+    assert app._get_enable_scheduler_step(None) == expected_step
 
 
-@pytest.mark.parametrize("units_slice", [1, 2, 3])
-def test_nova_compute_get_disable_scheduler_step(model, units_slice):
+@pytest.mark.parametrize(
+    "units",
+    [
+        [f"nova-compute/{unit}" for unit in range(1)],
+        [f"nova-compute/{unit}" for unit in range(2)],
+        [f"nova-compute/{unit}" for unit in range(3)],
```

Comment:
not-blocking: I think it would be ok to define without range, but definitely no blocker.